test/controllers/organization_controller_test.exs

```diff
@@ -0,0 +1,109 @@
+defmodule CodeCorps.OrganizationControllerTest do
+  use CodeCorps.ConnCase
+
+  alias CodeCorps.Organization
+  alias CodeCorps.Repo
+  alias CodeCorps.SluggedRoute
+
+  @valid_attrs %{description: "Build a better future.", name: "Code Corps"}
+  @invalid_attrs %{}
+
+  setup do
+    conn =
+      %{build_conn | host: "api."}
+      |> put_req_header("accept", "application/vnd.api+json")
+      |> put_req_header("content-type", "application/vnd.api+json")
+
+    {:ok, conn: conn}
+  end
+
+  defp relationships do
```

Comment:
I'm not sure if this is a real elixir style thing, but most of the time when I see a one liner it's formatted in one line:

```elixir
defp relationships do: %{}
```